conda/recipes/ucxx/recipe.yaml

```diff
@@ -0,0 +1,323 @@
+        - ${{ compiler("cxx") }}
+        - ${{ compiler("cuda") }}
+        - cmake ${{ cmake_version }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+          else: cuda-cudart-dev
+        - libucxx
+      run:
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+      ignore_run_exports:
+        from_package:
+          - cuda-cudart-dev
+          - python
+          - ucx
+        by_name:
+          - cuda-version
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: libucxx tests executables
+
+  - package:
+      name: ucxx
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_py${{ py_version }}_${{ date_string }}_${{ head_rev }}
+      dynamic_linking:
+        overlinking_behavior: "error"
+        missing_dso_allowlist:
+          - "lib/libpython3.12.so*"
+      script:
+        content:
+          - cmake --install cpp/python/build
+          - ./build.sh ucxx --cmake-args="\"-DFIND_UCXX_PYTHON=ON\""
+        secrets:
+          - AWS_ACCESS_KEY_ID
+          - AWS_SECRET_ACCESS_KEY
+          - AWS_SESSION_TOKEN
+        env:
+          CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER") }}
+          CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER") }}
+          CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER") }}
+          CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR") }}
+          PARALLEL_LEVEL: ${{ env.get("PARALLEL_LEVEL") }}
+          SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET") }}
+          SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT") }}
+          SCCACHE_REGION: ${{ env.get("SCCACHE_REGION") }}
+          SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL") }}
+          SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS") }}
+          SCCACHE_S3_KEY_PREFIX: libucxx-${{ env.get("RAPIDS_CONDA_ARCH") }}
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ninja
+        - ${{ compiler("c") }}
+        - ${{ compiler("cxx") }}
+        - ${{ compiler("cuda") }}
+        - cuda-version =${{ cuda_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - python
+        - pip
+        - scikit-build-core >=0.10.0
+        - cython >=3.0.0
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+        - ucx
+        - rapids-build-backend >=0.3.0,<0.4.0.dev0
+        - rmm ${{ rapids_version }}
+        - cuda-version =${{ cuda_version }}
+        - pynvml >=12.0.0,<13.0.0a0
+      run:
+        - python * *_cpython
```

Comment:
I think this constraint (`*_cpython`) is unnecessary. Having `python` in `host` exports a `python_abi` constraint that I think covers this. I see we're ignoring that below but that might be a mistake.